### ISSUE TITLE
#normalizing data base

### ISSUE THREAD

**Issue by hemanth-reddy-git** [Tec4Tric/Handwritten-Digit-Recognition#1]
in fist line it showing attribute error
keras.utils

**Comment by Tec4Tric** [comment]
Hi, thanks for letting me know the issue.
But, on my end the code is running absolutely fine, maybe it is a version issue.
If you can send me a screenshot or paste the exact error you are getting, I might help you then.
Thanks.
